riptide-stream/src/test/java/org/zalando/riptide/stream/EnforceCoverageTest.java

```diff
@@ -37,14 +37,14 @@
 
     @Test
     public void shouldUseStreamsConstructor() {
-        new Streams();
+        assertNotNull(new Streams());
     }
 
     @Test
     public void shouldUseStreamConverterConstructor() {
-        Streams.streamConverter();
-        Streams.streamConverter(null);
-        Streams.streamConverter(null, null);
+        streamConverter();
```

Comment:
My test do. And they implicitly check the defaults, too. Only, the above public support functions are tested separately. The best I would offer for them is checking that call are not throwing unexpected exceptions and that the result is an actual object.